src/atoms/Button.js

```diff
@@ -17,7 +17,7 @@ const Button = styled.button`
 	position: relative;
 	text-decoration: none;
 	text-align: center;
-	border-radius: 0;
+	border-radius: ${props => props.borderRadius || 0}%;
```

Comment:
maybe use defaultProps? :)

Reply:
+1

Reply:
Include the percentage sign in the prop value as well.